client/src/pages/ProfileCompleted.js

```diff
@@ -16,13 +17,22 @@ import {
 import GTM from "constants/gtm-tags";
 import { connect } from "react-redux";
 
-const ProfileCompleted = ({ user, refetchUser }) => {
+const ProfileCompleted = ({ user, refetchUser, setOrganisationId }) => {
   const { t } = useTranslation();
   const location = useLocation();
   useEffect(() => {
     refetchUser();
     // will only run once (on mount) to update navbar org list
   }, [refetchUser]);
+
+  const switchAccountDrawerMenu = () => {
+    if (location?.state?.orgId) {
+      const index = location.state.orgId;
```

Comment:
I am only not sure about this part.
Does the router state have the newly created orgId?

Reply:
Yeap, below sample from `CreateOrganisationProfile` 

![code](https://user-images.githubusercontent.com/34690158/103635038-fcaf5f00-4f58-11eb-8b49-8306479d31d6.png)